.editorconfig

```diff
@@ -27,6 +26,10 @@ indent_size = 2
 indent_style = space
 indent_size = 2
 
-[target/start-mailserver.sh]
```

Comment:
What happens with start-mailserver.sh? Do we use 2 spaces now, or is it not checked at all?

Reply:
With this change `start-mailserver.sh` is simply treated like all the other bash scripts in this repository.  But even with the exception for start-mailserver there were quiet a lot of lines that would have needed fixing (did not count).

Reply:
Yes, I've noticed.